src/routes/blogs/+page.svelte

```diff
@@ -45,6 +45,16 @@
 		dispatch('switchTab', tab);
 	}
 	let featuredblog = [
+		{
+			title: 'Enhancing team collaboration during AI model optimization with the Olive Shared Cache',
+			date: 'October 30th, 2024',
+			blurb:
+				"Learn how to use Olive's shared cache to enhance team collaboration when optimizing AI models",
+			link: 'blogs/olive-shared-cache',
+			image: Phi3SmallMediumImage,
```

Comment:
Needs to be a blog post image (also update actual page with image)

Reply:
I've added the ONNX Logo. We don't have an Olive logo.